app/src/validation/post.js

```diff
@@ -22,10 +22,32 @@ const validatePost =async function (post) {
     if( await postModel.isSlugExist(post)) {
         error.push('The slug already exists');
     }
-   
+    let sign = post.slug;
+    if (hasForbiddenSigns(sign)) {
+        error.push('Slug can not contain polish signs'); 
+    }
+
     return error;
 };
+/**
+ * @param {string} post 
+ * @returns {boolean} 
+ */
+function hasForbiddenSigns(sign) {
+
+    if (typeof sign == 'string') {
+        let checkSigns = sign.split('');
+        for (let i = 0; i < checkSigns.length; i++) {
+            if (checkSigns[i] === 'ą' || checkSigns[i] === 'ę' || checkSigns[i] === 'ć' || checkSigns[i] === 'ó' || checkSigns[i] === 'ź' || checkSigns[i] === 'ż') {
+                return true;
+            }
+        }
+    }
+
+    return false;
```

Comment:
Jeśli sign nie będzie strigniem to funkcja zwróci false

Reply:
+ brak testów do funkcji

Reply:
na odwrót :P